docs/configuration.md

```diff
@@ -40,13 +40,16 @@ This option obviously would not make sense for language servers for other langua
 
 Here is a list of the additional settings currently supported by `haskell-language-server`, along with their setting key (you may not need to know this) and default:
 
-- Formatting provider (`haskell.formattingProvider`, default `ormolu`): what formatter to use; one of `floskell`, `ormolu`, `fourmolu`, `stylish-haskell`, or `brittany` (if compiled with the brittany plugin)
-- Format on imports (`haskell.formatOnImportOn`, default true): whether to format after adding an import
-- Diagnostics on change (`haskell.diagnosticsOnChange`, default true): (currently unused)
-- Completion snippets (`haskell.completionSnippetsOn`, default true): whether to support completion snippets
-- Liquid Haskell (`haskell.liquidOn`, default false): whether to enable Liquid Haskell support (currently unused until the Liquid Haskell support is functional again)
-- Hlint (`haskell.hlintOn`, default true): whether to enable Hlint support
+- Formatting provider (`haskell.formattingProvider`, default `ormolu`): what formatter to use; one of `floskell`, `ormolu`, `fourmolu`, `stylish-haskell`, or `brittany` (if compiled with the brittany plugin).
+- Format on imports (`haskell.formatOnImportOn`, default true): whether to format after adding an import.
+- Diagnostics on change (`haskell.diagnosticsOnChange`, default true): (currently unused).
+- Diagnostics debounce duration (`haskell.diagnosticsDebounceDuration`, default 350000 milliseconds).
```

Comment:
Would like to add more info about this but i am not sure about its semantics

Reply:
This isn't used anywhere, rather than documenting it we should delete it

Reply:
deleting here for now

Reply:
I insist on deleting it from code as well, otherwise future maintainers will be puzzled as well:

https://github.com/haskell/haskell-language-server/blob/d815d0474d226e00ba2e3f93f9878b109e1ea2e0/hls-plugin-api/src/Ide/Plugin/Config.hs#L55